.env.default

```diff
@@ -0,0 +1,8 @@
+WEBEX_CLIENT_ID=
+WEBEX_CLIENT_SECRET=
+IDBROKER_BASE_URL=
```

Comment:
can you please add default values here? Except for client id, secret, access token and destination.